packages/one-app-runner/README.md

```diff
@@ -0,0 +1,257 @@
+# @americanexpress/one-app-runner
+
+> A command line interface (CLI) tool for running [One App](https://github.com/americanexpress/one-app) locally.
+
+**one-app-runner** allows you to work locally on your [Holocron Module](https://github.com/americanexpress/one-app/blob/master/docs/api/API.md#modules) without having to clone [One App](https://github.com/americanexpress/one-app).
```

Comment:
should we mention something about watching your builds and whether you need to restart one-app-runner?

Reply:
yeah probably should.  We also have #38 open.

Reply:
Fixed in latest commit!